noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -129,7 +129,7 @@ pub global INITIALIZATION_SLOT_SEPARATOR: Field = 1000_000_000;
 pub global INITIAL_L2_BLOCK_NUM: Field = 1;
 pub global PRIVATE_LOG_SIZE_IN_FIELDS: u32 = 18; // This is currently affected by the size of the log overhead defined in aztec-nr/aztec/src/encrypted_logs/payload.nr.
 pub global BLOB_SIZE_IN_BYTES: Field = 31 * 4096;
-pub global AZTEC_MAX_EPOCH_DURATION: u32 = 32;
+pub global AZTEC_MAX_EPOCH_DURATION: u32 = 48;
```

Comment:
lgtm if no further propagation needed, curious how this affects circuit compilation